pennylane/ops/functions/__init__.py

```diff
@@ -13,6 +13,16 @@
 # limitations under the License.
 """
 This module contains functions that act on operators and tapes.

```

Comment:
Do you think it might be confusing having separate `ops/functions` and `ops/op_math` going forward? There might be edge cases where it's not clear to a developer which file to add the feature to.

Reply:
It's already confusing.  The `simplify` function could go in either.

Reply:
Agree - should we just merge them? Have a single place for generic functions of operators?

Reply:
We tend to group things in PennyLane by what type of object they are: functions, classes, tranforms, etc.  We should instead start grouping things by dependencies, interactions, and responsibilities. But there's no good, clear cut answer on where to put things, especially in a language as flexible as Python.

Reply:
Yep for sure, it is not an easy problem to solve. My short term thinking was to avoid developer overhead that we already see with operations vs. templates ("should I add this to operations or templates?")

Reply:
Re-working the docs might be a good epic to add in the future since I imagine this will require moving multiple functions and classes around.